src/config.rs

```diff
@@ -353,6 +347,34 @@ impl GitHub {
     }
 }
 
+/// Take a quoted string from the config file and unescape it (i.e. strip the start and end quote
+/// (") characters and process any escape characters in the string.)
+fn unescape_str(us: &str) -> String {
+    // The regex in config.l should have guaranteed that strings start and finish with a
+    // single quote (") character.
```

Comment:
A single quote. Shows a double quote in brackets. Hilarity ensues :)

Reply:
Huh?

Reply:
> The regex in config.l should have guaranteed that strings start and finish with a single quote (") character.

But there's a double quote in the brackets.

I suspect you meant to say double quote?

Reply:
Oh, I see! It means "single double quote" if that makes sense?

Reply:
I see. I never would have gotten that from the prose. "single double quote" also offers the reader the suspicion that there may be a typo.

I think we should be really captain obvious about it and say:
> ...should have guaranteed that strings start and finish with a double quote (") character.

Reply:
I agree: the phrasing was pretty terrible! Fixed in https://github.com/softdevteam/snare/pull/44/commits/7c6d4032bc4c3ae415d5f57099fc6c8d3b7e8ace.